docs/developer-docs/latest/developer-resources/database-migrations.md

```diff
@@ -30,23 +34,33 @@ The `up()` function runs in a database transaction which means if a query fails
 There is no CLI to manually execute the database migrations.
 :::
 
-## Creating a migration file
+## Create a migration file
 
-To create a migration file:
+Use the following procedure to create a migration file:
 
-1. In the `./database/migrations` folder, create a new file named after the date and the name of the migration (e.g. `2022.05.10T00.00.00.name-of-my-migration.js`). Make sure that the file name follows this naming pattern, because the alphabetical order of the files defines the order in which the migrations have to run.
+1. In the project root directory, run the command:
 
-2. Copy and paste the following template in the previously created file:
+    <code-group>
+    <code-block title="YARN">
 
-```jsx
-'use strict'
+    ```bash
+    yarn strapi generate migration
+    ```
 
-async function up(knex) {}
+    </code-block>
+    <code-block title="NPM">
 
-module.exports = { up };
-```
+    ```bash
+    npm run strapi generate migration
+    ```
+
+    </code-block>
+    </code-group>
+
+2. Type a filename when prompted. The output filename is a concatenation of a date-time string and your filename with the format: `YYYY.MM.DDTHH.MM.SS.name-of-my-migration.js`
```

Comment:
```suggestion
2. Type a migration name when prompted. The output filename is a concatenation of a date-time string and your filename with the format: `YYYY.MM.DDTHH.MM.SS.name-of-my-migration.js`
```

Reply:
Do we allow configuration of this format somewhere? I've never seen that date format used with periods.

Typically it should be `YYYY-MM-DD_HH:MM:SS_name-of-file.js`